src/coreclr/vm/threadsuspend.cpp

```diff
@@ -5733,7 +5733,9 @@ BOOL CheckActivationSafePoint(SIZE_T ip)
 
     // The criteria for safe activation is to be running managed code.
     // Also we are not interested in handling interruption if we are already in preemptive mode.
-    BOOL isActivationSafePoint = pThread != NULL &&
+    // Also we are not interested in handling interruption if we are single stepping
```

Comment:
Nit: Maybe consider simplifying the three lines of comments to: 
// The criteria for safe activation is to be running managed code.
// Not interested in handling interruption if we are already in preemptive mode nor if we are single stepping